src/api/app/models/token/errors.rb

```diff
@@ -12,4 +12,8 @@ class NonExistentWorkflowsFile < APIError
   class SCMTokenInvalid < APIError
     setup 401
   end
+
+  class WorkflowsYamlNotParsable < APIError
```

Comment:
This error isn't really related to the `Token` models. Maybe `src/api/app/models/workflow/errors.rb` would be a better location. That file doesn't exist yet, so it needs to be created.

Reply:
yes, later maybe